### ISSUE TITLE
New startsplash

### ISSUE THREAD

**Issue by comradekingu** [hvdwolf/jExifToolGUI#135]
This is the current situation for 1.8.1
![bilde](https://user-images.githubusercontent.com/13802408/130359232-a61d6448-7e78-4fed-bb54-4f4173e79a7b.png)

Should be "GPS" and "XMP".
Other such issues.

**Comment by hvdwolf** [comment]
Just corrected the splash logo and committed to repo. I also removed the "bold" font for the Exif, GPS and XMP.

![jexiftoolgui-splashlogo](https://user-images.githubusercontent.com/4520361/130360998-33bd6844-2037-4acd-abe0-123c57db99d7.png)

**Comment by mrtngrsbch** [comment]
Hummm...
Coming soon a contest for the logo v2?

**Comment by comradekingu** [comment]
The "It would be hard to point a camera at anything worse"-competition :)
Who will make the exifitive decision?
I can't decide and participate, and you should be disqualified :)))

**Comment by hvdwolf** [comment]
The logo was first used for my pyExifToolGUI out of 2011 (2012 to Github).
First of all:  I'm not a programmer, and secondly: I'm not a designer. I build something because there was (at that time) not something open source that could do the same. And yes: then this piece of software needs an icon and splash logo.

But is it really so bad? _"It would be hard to point a camera at anything worse"_
If someone can come up with a better logo: be my guest.

**Comment by mrtngrsbch** [comment]
I personally have a certain attachment to this "retro" logo.
But I think it lacks the essence of the functions it has.
I think it would put more emphasis on the power of metadata.
I would launch the logo change with the diffusion of weblate and a big request to the community and take advantage of the generated inertia, but without any requirement to change it if the results are not as expected.

Although the most important thing, for me and @gregreser , is to be able to work in the interface.

**Comment by comradekingu** [comment]
It isn't bad, but it is just so generic to that period.
I think a "behind the scenes" image of something breaking first into an animated complexity and then something data-like would do.
Can functionality be reconsidered in terms of advertising for something the user already has installed, and decided to open?
This is from my phone, as an example
![image](https://user-images.githubusercontent.com/13802408/130454094-2f3d428c-0d4d-43e6-9487-6ca196a9bd04.png)

**Comment by hvdwolf** [comment]
What is "Exiftive"?
And I like the image of the boat, but from photographers perspective the dynamic range fails short and the lighter parts are blown out, missing any detail. Maybe one would not even see that in a small splash screen, but those are details I am looking at.

The name jExifToolGUI was chosen for a reason. As mentioned, the previous python/pySide program was pyExifToolGUI. I heard from several users mentioning the name was complicated or too difficult, but at the same time never forgot it anymore. There was a review in a french magazine exactly saying that. I wanted the same for jExifToolGUI, but maybe there is a more catchy name possible.

**Comment by mrtngrsbch** [comment]
jExifToolGUI is a complex and unique name. Changing it after +10 years I don't think it is relevant unless we want to sell it to a large corporation.
What I don't like about the current logo is that the landscape with a tree looks like a windows wallpaper, but it still makes me laugh.

If I were to make some minor changes, I would limit myself to the typography and texts. I would incorporate "VRA Core", just to emphasize the professional orientation to archives.

Anyway, for the moment I see it secondary to restrain energies on this topic.

@hvdwolf if you send me the original logo maybe I can play with it a little bit.

best

**Comment by MetadataDeluxe** [comment]
jExifToolGUI is a bit long, but it explains what it app does and is memorable. I think it would be a mistake to change it.
The changes you made to the logo (GPS, XMP, and removing bold) are better. I agree with @mrtngrsbch that this is not the most important thing to spend time on.
Actually, I think the logo works very well as a small taskbar icon. It's unique and easily recognizable.
![image](https://user-images.githubusercontent.com/5862936/130531509-c3859b77-fefb-42ed-8174-408c29159d19.png)

**Comment by mrtngrsbch** [comment]
Possibly we can work on a splash screen + a logo.
but I think we have already had this conversation with @hvdwolf and some Java limitations, that I don't remember now.

As for the tree... I think it would be nicer to see it bigger and without text in the taskbar.

**Comment by hvdwolf** [comment]
I don't want to mention all metadata options, as this is really dependent on what the user the user wants. I think that less than 1% of the users even know what ISAD(G) or VRA Core means. Of course: if this will be only one of the very few tools that do support isadg and vra that precentage might grow ;)
Yesterday evening I created another logo, very quick&dirty, (currently maybe even uglier), describing very general what it does.
For me it is now more about the text. Is this a more suitable text?

![image](https://user-images.githubusercontent.com/4520361/130633713-7e94da60-a7f1-41b2-b109-00f841bbd9db.png)

**Comment by comradekingu** [comment]
Gave me an idea. What about a camera pointed at the user, and then the magnifying glass is the lens looking into it?
I think that can be made to look different from a lens-shifter if the colour is different.

**Comment by hvdwolf** [comment]
> 
> For an icon, what about a camera pointed at the user, and then the magnifying glass is the lens looking into it?
> 
Something like this? But then "iconized"
![image](https://user-images.githubusercontent.com/4520361/130636315-ae3a9609-27f3-48e3-a4b1-ffe19b93c219.png)

**Comment by comradekingu** [comment]
I was thinking something like
![bilde](https://user-images.githubusercontent.com/13802408/130643764-a2d2a379-9a7c-47a6-8fdc-aaa6a05b56fc.png)

**Comment by mrtngrsbch** [comment]
I liked this because it focuses on "metadata tagger" and the image is very technical and less pictorial.
"The image metadata tagger" sounds like an excellent title to me.

PS: But be careful Harry... one day archivists and museologists will take over the world and they will all be under our boots ;-)


> I don't want to mention all metadata options, as this is really dependent on what the user the user wants. I think that less than 1% of the users even know what ISAD(G) or VRA Core means. Of course: if this will be only one of the very few tools that do support isadg and vra that precentage might grow ;)
> Yesterday evening I created another logo, very quick&dirty, (currently maybe even uglier), describing very general what it does.
> For me it is now more about the text. Is this a more suitable text?
> 
> ![image](https://user-images.githubusercontent.com/4520361/130633713-7e94da60-a7f1-41b2-b109-00f841bbd9db.png)
> 
> @mrtngrsbch : The image is in the reppository as Gimp "jexiftoolgui-splashlogo.xcf" inside images.
> The splashlogo itself is "/src/main/resources/icons/jexiftoolgui-splashlogo.png"

**Comment by mrtngrsbch** [comment]
Hi @hvdwolf ,
Here we have been thinking about the logo and we have some questions:

Can we create a splash screen and a different logo for the icon bar?
The comment from @MetadataDeluxe seems important to me [https://github.com/hvdwolf/jExifToolGUI/issues/135#issuecomment-904200534]

Furthermore...
We liked this font: https://fonts.google.com/featured/Plex
and something similar to this for the icon bar

![2042146](https://user-images.githubusercontent.com/693328/131699652-497c9df5-3b2a-42f9-9fd1-50b873995d93.png)

**Comment by mrtngrsbch** [comment]
I love that lens "made in Paraguay" ;-)

obviously I wouldn't use the texts on the lens... it's just illustrative.

**Comment by hvdwolf** [comment]
Are the images copyright free or Creative Commons or some other open license?

**Comment by comradekingu** [comment]
I just picked some camera icon to start with and drew a magnifying glass on that, but it can be redrawn easily.
I tried to find the lens since I don't know of any lenses made in Paraguay https://yemilo.com/
"Aperture laboratories" and others use similar logos to the last one.

**Comment by hvdwolf** [comment]
It could be something like this, but seeing it on that website, I'm afraid it is copyrighted.
![newLens2Splash](https://user-images.githubusercontent.com/4520361/131716990-333877fc-c762-42ae-a14f-4cae617a76a3.png)

**Comment by mrtngrsbch** [comment]
> Are the images copyright free or Creative Commons or some other open license?

Yes, obviously we will use free licensed images.

**Comment by mrtngrsbch** [comment]
> It could be something like this, but seeing it on that website, I'm afraid it is copyrighted.

wait @hvdwolf... this image I have sent is ONLY an example for you to understand the way to follow. Just an example and NO texts or marks

**Comment by mrtngrsbch** [comment]
> I just picked some camera icon to start with and drew a magnifying glass on that, but it can be redrawn easily.
> I tried to find the lens since I don't know of any lenses made in Paraguay https://yemilo.com/
> "Aperture laboratories" and others use similar logos to the last one.

Hi @comradekingu, "Made in Paraguay" is a joke of some comic guy. Indeed Paraguay never manufactured a lens ;-)
I just sent the image for illustrative purposes.

**Comment by MetadataDeluxe** [comment]
The Noun Project is a great source for Creative Commons icons
https://thenounproject.com/search/?q=lens

**Comment by mrtngrsbch** [comment]
Cool....
and this: https://wallpaperaccess.com/
and this: https://www.vectorstock.com/

We just need a few days without haste to make something interesting.
My brother and I believe that the JTG icon should have these ingredients, but with colors

![photo_2021-09-01 12 33 48](https://user-images.githubusercontent.com/693328/131766531-aced219e-c7bf-4036-9a05-64f03458e8d8.jpeg)

**Comment by MetadataDeluxe** [comment]
I like the concept of camera lens iris and a tag filled with metadata

**Comment by mrtngrsbch** [comment]
Hi guys,

I started to play with the ideas ;-)

![image](https://user-images.githubusercontent.com/693328/139556365-243e0047-528d-4f57-ad1e-16ca3f0407da.png)

**Comment by mrtngrsbch** [comment]
other version...

![image](https://user-images.githubusercontent.com/693328/139557014-a91dbd2e-8ff5-4015-8dc2-a005795e22e7.png)

**Comment by hvdwolf** [comment]
Sorry. I know my own icon looks like the Win95 landscape, but that I don't want anymore.
I do like the the fade-in of the binary numbers.
I like the 2nd one, but then with another background, but to say "The best GUI ..". I'm not sure.

**Comment by mrtngrsbch** [comment]
Hi... 
OK for the landscape background...
"the best" is a momentaneous joke ;-)

the binary numbers are from demo > they should be metadata, but I have to do that 'by hand'.

**Comment by mrtngrsbch** [comment]
Update in honor of Bruegel & Netherland
https://www.metmuseum.org/art/collection/search/435809


![image](https://user-images.githubusercontent.com/693328/139557736-34fbe3c9-117a-4044-a3ea-1656c25783fa.png)

**Comment by mrtngrsbch** [comment]
I'm making these sketches to get closer to a formal idea.
Then the elements can be changed easily, like texts and images.
The main thing is that we are happy with the splashscreen mockup.

**Comment by comradekingu** [comment]
I like how friendly the second one is. Pair it with a better more simple image.
I think some structures with the text is better. Like roots from a tree or similar.
Just the blue with a little avatar next to the text?

**Comment by mrtngrsbch** [comment]
> I like how friendly the second one is. Pair it with a better more simple image. I think some structures with the text is better. Like roots from a tree or similar. Just the blue with a little avatar next to the text?

Hi @comradekingu,
I just put an image of Brueguel to send a message to Harry ;-)
Honestly I don't have a specific image and we can change it with each version, if we like it.
It's true that a Brueguel's work is something very particular and maybe a common image would be better. 
Do you have any idea ? https://unsplash.com/ ?

I am also a photographer and I have many of mine at: https://martin.gersbach.net/

Regarding the text with structures... yes, it's just a first sketch to talk about. Then we can add aesthetic details. 

Regarding the Icon... we haven't incorporated your suggestions above yet, I hope to do it soon. In the meantime we are playing with the ideas

best

**Comment by mrtngrsbch** [comment]
another version...

![image](https://user-images.githubusercontent.com/693328/139564572-ee7e12fc-96ed-468c-920e-2827368930e4.png)

**Comment by hvdwolf** [comment]
You mean something like this?

![new-splashlogo](https://user-images.githubusercontent.com/4520361/139571395-c0de720d-2594-406b-b89e-267a11415577.jpg)
The text can be  made more or less transparent.

**Comment by hvdwolf** [comment]
Ah! :)
You already changed the text. My preference for "subtitle" would be "A graphical frontend for Phil Harvey's ExifTool"

**Comment by MetadataDeluxe** [comment]
I like the winter landscape version.

One possible problem with the logo - it's similar to Darktable (except for the tag)
https://www.darktable.org/
![image](https://user-images.githubusercontent.com/5862936/139730213-381a5036-66d7-4af9-98c9-6016673d83f9.png)

**Comment by MetadataDeluxe** [comment]
I like using real metadata beneath the landscape. It would be even better if it included title, creator, subject etc. of the landscape (or whatever image is used). 

I also like "A graphical frontend for Phil Harvey's ExifTool"

**Comment by mrtngrsbch** [comment]
> Ah! :) You already changed the text. My preference for "subtitle" would be "A graphical frontend for Phil Harvey's ExifTool". I looked to much at the graphics.
> 
> **Note:** With the new way of packaging the app with packr on macOS, you would loose the splashscreen. That is until packr fixes that bug, but it seems it is something between java and macOS that they can't influence.

oh no !
bad news !!!
So I'm freezing the splashscreen project and we're going to revise the logo issues until further notice.

**Comment by mrtngrsbch** [comment]
> I like the winter landscape version.
> 
> One possible problem with the logo - it's similar to Darktable (except for the tag)

Yes, it is true that it is similar. This can be positive as well as negative... however, we will review this point.

**Comment by mrtngrsbch** [comment]
> I like the winter landscape as well.
> 
> Or? ![new-splashlogo](https://user-images.githubusercontent.com/4520361/139571395-c0de720d-2594-406b-b89e-267a11415577.jpg) The text can be made more or less transparent. I simply copied it in.

yes... the winter landscape is very nice and you can change the seasons of the year with the new versions.
The metadata below the image are for illustrative purposes ONLY, not the real ones that I would like to put as well.

**Comment by mrtngrsbch** [comment]
> I like using real metadata beneath the landscape. It would be even better if it included title, creator, subject etc. of the landscape (or whatever image is used).
> 
> I also like "A graphical frontend for Phil Harvey's ExifTool"

Yes Greg, as soon as I have the main model assembled we will proceed to improve the quality of every aspect, such as the actual metadata.
OK for the description.

**Comment by mrtngrsbch** [comment]
here another minor version... (with a minor bug in layer ;-)
![image](https://user-images.githubusercontent.com/693328/139784947-fe774c4b-dde4-449d-9f56-9f6ed143548e.png)

@hvdwolf, Creo que es mejor una imagen 'real', de un fotografo. Usar paisajes quizá sea un buen recurso. Yo tengo muchas fotos de paisajes y estaria contento de ofrecerlas y llenarlas de metadatos diversos.
https://martin.gersbach.net/work/

@MetadataDeluxe,  nuevo texto incluido

@all, muy triste la noticia que de packr no muestra el splashscreen in MacOS ! Mismo así pdemos seguir con el diseno.

**Comment by mrtngrsbch** [comment]
et voilà, here another version

![image](https://user-images.githubusercontent.com/693328/139788053-efd1fbf0-8383-4a2b-bd2b-8649ce8111c5.png)

**Comment by MetadataDeluxe** [comment]
That metadata list is easy to read, but I like the older version where the text fills the entire area below the landscape. Maybe the metadata should be hard to read - emphasizing that jExifToolGUI brings order to chaos.

**Comment by mrtngrsbch** [comment]
Yes, it is true what you say but this is already of the aesthetic order. 
Then we will have to keep looking for alternatives that transmit well the message of <metadata>.

Of course, for the final version, it occurs to me that the metadata displayed should be relevant and varied (GPS, VRA, IPTC...), not a simple copy & paste.

demo:
![image](https://user-images.githubusercontent.com/693328/139901102-c2b9d3c1-48ec-4c85-a28d-b09891ccfacc.png)

**Comment by comradekingu** [comment]
@mrtngrsbch I like
https://martin.gersbach.net/canon-del-rio-azul-16/
https://martin.gersbach.net/canon-del-rio-azul-08/ and maybe
https://martin.gersbach.net/canon-del-rio-azul-14/

![bilde](https://user-images.githubusercontent.com/13802408/139905481-e4fc7ccd-4523-45f0-9709-0005e4348737.png)
A graphical front-end
for Phil Harvey's ExifTool

Edit, browse and write metadata.
Format1
Format 2
Format3
Format 4

Copylefted libre software,
licensed GPLv3+
Use, study, copy and share; with all.

Keep the space between elements.
Maybe some semi-transparancy on the bottom right?

**Comment by mrtngrsbch** [comment]
![image](https://user-images.githubusercontent.com/693328/139911000-e8c8b554-dd41-40f0-9b30-d962d6a6947b.png)

**Comment by hvdwolf** [comment]
I like the last one and the few before that as well. One question: The version is now in the png. That requires an update of the png for every release (and yes, I know there was a lot of time between 1.8.1 and 1.9.0).


> 
> @ALL, I am very sad to hear that packr does not show the splashscreen in MacOS ! Even so we can continue with the design.

packr has one other drawback. It only allows for full bundles with java included.
AdoptOpenJDK (Temurin versions) version 14+ and up now comes with the revived/recreated jpackage from the old Sun java V7 version. I will also take a look at that, but did not do that yet. The only issue with jpackage is that I need to create the package really on MacOS, while my "own method" and packr are cross-platform. You can do that on any OS, which means in my case from Linux. And I do not know what jpackage will do with the splashscreen as the packr guys claim this to be an issue between java and macOS.
Java V11 is/was the LTS version and is still long supported. java V16 has just been appointed as the new LTS version.

**Comment by mrtngrsbch** [comment]
I have no problem to make changes to each version, leave several versions ready and also send you the full PSD to GitHub or send it to you.

Currently it is PNG because I have only sent silly screenshots until I have a final version.

packr vs. jpackage = packr
I see that the limitation of jpackage is much more serious !

Personally I have little interest in splashscreens, but in this case I am attached to it ;-)

We still need to:
- Work on the ICON and review the previous comments
- Improve the metadata layer
- Last final revisions

**Comment by hvdwolf** [comment]
jpackage also only builds a full package. And jpackage doesn't show the splash screen either.
So this splash screen issue has indeed nothing to do with packr.
And if you use the jar, you can still see the splash screen :wink:

I work with gimp in xcf, but a PSD is fine as well. The final splash screen has to be a png anyway as java can't handle .psd and .xcf

**Comment by mrtngrsbch** [comment]
Hi guys, @comradekingu @MetadataDeluxe @hvdwolf 

We have new isotype & splash screen updates. 
A little bit of work and we are getting closer to a final version.

note: we consider that it would be interesting to use images with public license and of the same theme common to all photographers. In this case we chose animals ;-)

Enjoy 



![photo_2022-01-18 11 01 46](https://user-images.githubusercontent.com/693328/149951863-9ab083e3-74cf-4e35-9475-a947c1b590ce.jpeg)
![photo_2022-01-18 11 01 28](https://user-images.githubusercontent.com/693328/149951883-91a4dce5-37e2-4163-aa17-ef52599ba12d.jpeg)
![photo_2022-01-18 11 01 35](https://user-images.githubusercontent.com/693328/149951881-c2167a0d-58db-417b-9f2f-f4bf2c05e82d.jpeg)

![photo_2022-01-18 11 01 19](https://user-images.githubusercontent.com/693328/149951918-1421ee8e-abd5-41ad-8cae-69b3b9824425.jpeg)


![photo_2022-01-18 11 01 14](https://user-images.githubusercontent.com/693328/149951925-e6980ffb-820c-4869-8fe4-d596e5f8504b.jpeg)

![photo_2022-01-18 11 06 57](https://user-images.githubusercontent.com/693328/149952615-cc5ea35d-b9ce-4439-8e8e-afcee0cb3471.jpeg)

**Comment by mrtngrsbch** [comment]
@hvdwolf, 
Please tell me where to upload the set of images (PSD & PNG) for a first public beta version (v5)

**Comment by mrtngrsbch** [comment]
well...
I expect this version is a good candidate to be considered as 1st release.

## icon/isotype
![jExifTool-ico](https://user-images.githubusercontent.com/693328/150246778-d8d8501b-7824-40d4-9afc-13f34dad60e8.png)

## splash
![jExifToolGUI-splash-v4_sebasg](https://user-images.githubusercontent.com/693328/150246869-d522e251-a33a-401d-ac00-1e7c5cd1b32b.png)

**Comment by comradekingu** [comment]
I like how it is eying the logo, ready to snap it up.

**Comment by mrtngrsbch** [comment]
Thanks a lot for your comment, we have put a lot of love in this work. ;-)

We have used the color pentaprism as a wheel, like many photo apps (Mac & Google photo, etc). 
It is more evolved than the Darktable isotype (which is very basic and found in many image libraries), but still retains a similarity that makes it belong to the same family of apps.
We have tested it in the Mac OS menu and it looks great; it stands out from the rest of the apps. We have also tested the black and white variant.

**Comment by MetadataDeluxe** [comment]
I like this new icon and the splash screen design very much.

**Comment by hvdwolf** [comment]
Hi really like the image as well. 
Can you make a version 2.0.0? That will be the new version. I made this version jump because of the new grid layout.
I shortly want to release the new version. It is not bullet proof yet, but I  am currently in a world-wide mega project taking a few more months and my work times are currently sky high, incl. 30% of the weekends. 
There are enough changes for a new release. The rest has to wait.

**Comment by mrtngrsbch** [comment]
Hi,
Well... here is a new version ( jExifToolGUI 2.0 **Frog**  ;-) with small corrections (metadata layer, iso, title placement...)
Tell me how you prefer me to send you the files. I do directly a commit ?

Also we are working with the icons in different sizes (16, 32, 48, 256...)
![jExifToolGUI-splash-v4-1_marting](https://user-images.githubusercontent.com/693328/150568637-504e2f4c-e121-4060-9f46-400486795ac7.png)

Once we validate this "frog" version we can keep the image and just create clones with the sub-versions or also change the pictures > it is very easy (e.g. sub-versions 2 can be just frogs).

We must also add 2 credits:
1- iso/splash design (Sebastian Gersbach)
2- image (https://unsplash.com/s/photos/frog & others)

**Comment by hvdwolf** [comment]
I compiled a new 2.0.0-beta version using the frame icon and the splash screen and including the credits (Help -> Credits)
Note that due to Apples strange behavior when it comes to java bundles, the splashscreen doesn't show on MacOS. If you run the jar (java -jar jExifToolGUI.jar) you will see the splash screen.
Note that program and frame icons need to be square. The current logo is  256x233. If this becomes 256x256 I can also create necessary icons and  macOS iconpack myself.I could resize it an build it myself, but I leave that for now to the original artist. So no new program icons yet.
At mega.nz: https://mega.nz/folder/NRsSjbxD#Lfe_9HcM1TqCTcfb1APj4w

**Comment by mrtngrsbch** [comment]
cool... 
nice spash screen ;-)
and we are at version 2.0 = Congrats !

Logo > this weekend we check this sizing problem and return one OK

ps:
I have discovered a small problem that I think was present in the previous version:
when opening JTG the bottom menu does not appear complete.

https://user-images.githubusercontent.com/693328/150591463-929c8552-828a-4d58-aeb8-82f7e5e6cd5f.mp4

**Comment by hvdwolf** [comment]
Yes, I see what you mean. The application remembers its last size and and where the left/right separation bar is upon app closure.
If a new version comes with a slightly different "basic" screen size, or even the slighthest change in font size, you get this akward behaviour. And with the adaptations I do in the gui, I might indeed have dragged the window edge by accident. Even if it is only 5 pixels.
Now if you drag that bottom edge again so that you can see it, it should display correctly on next starts.

**Comment by hvdwolf** [comment]
Created a beta2.
- Resized the canvas in the logo to 256x256
- Created an ico for windows
- Created an iconpack for macOS
- fixed issue 219 (vrae.config reverse check)

link: https://mega.nz/folder/EIli3IjC#1FPJ4_CL7xl5dkluWOqBpQ

**Comment by mrtngrsbch** [comment]
My JTG icon in MacOS ❤️ 

![image](https://user-images.githubusercontent.com/693328/150697116-44bff296-1422-448b-9814-6e1ea87f404f.png)

**Comment by mrtngrsbch** [comment]
Hi @hvdwolf ?

Here is the latest version of the Icons (16, 32, 48, 256 px) with the corrected dimensions

[_icon (resources).zip](https://github.com/hvdwolf/jExifToolGUI/files/7954645/_icon.resources.zip)
![jExifToolGUI icon](https://user-images.githubusercontent.com/693328/151463396-640c6c27-7205-47da-aac2-4b394c7497cf.png)

**Comment by hvdwolf** [comment]
Thanks. I applied them to the repository.
For the time being this I do not need the AI and PSD versions. They look nice and there is nothing I want to change.
